task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskDescriptionEditor.vue

```diff
@@ -179,20 +195,23 @@ export default {
             self.inputVal = newData;
           },
           destroy: function () {
-            self.inputVal = '';
             self.editorReady = false;
+            self.inputVal = '';
           }
         },
       });
     },
+    hideDescriptionEditor() {
```

Comment:
useful method but not used in current component, can you please use it to hide Editor

Reply:
I did it